spec/features/bidder_interacts_with_auction_spec.rb

```diff
@@ -2,153 +2,205 @@
+    end
 
-  scenario "Viewing an open auction" do
-    current_auction, _bids = create_current_auction
-    visit auction_path(current_auction)
+    scenario "There is a closed auction" do
+      @auction = FactoryGirl.create(:auction, :closed)
 
-    expect(page).to have_content("Open")
-  end
+      visit '/'
 
-  scenario "Viewing a closed auction" do
-    closed_auction, _bids = create_closed_auction
-    visit auction_path(closed_auction)
+      within(:css, 'div.issue-list-item') do
+        within(:css, 'span.usa-label-big') do
+          expect(page).to have_content('Closed')
+        end
+      end
 
-    expect(page).to have_content("Closed")
-  end
+      expect(page).to_not have_content('Bid »')
+    end
+
+    scenario "There is an expiring auction" do
+      @auction = FactoryGirl.create(:auction, :expiring)
+
+      visit '/'
 
-  scenario "Viewing auction detail page and navigating to bid history" do
-    create_current_auction
+      within(:css, 'div.issue-list-item') do
+        within(:css, 'span.usa-label-big') do
+          expect(page).to have_content('Expiring')
+        end
+      end
 
-    visit auction_path(@auction.id)
+      expect(page).to have_content('Bid »')
+    end
 
-    number_of_bids = "#{@auction.bids.length} bids"
-    expect(page).to have_content(number_of_bids)
+    scenario "There is a future auction" do
+      @auction = FactoryGirl.create(:auction, :future)
 
-    click_on(number_of_bids)
+      visit '/'
 
-    h1_text = "Bids for \"#{@auction.title}\""
-    expect(page).to have_content(h1_text)
-  end
+      within(:css, 'div.issue-list-item') do
+        within(:css, 'span.usa-label-big') do
+          expect(page).to have_content('Coming Soon')
+        end
+      end
 
-  scenario "Viewing auction list and detail view as a logged out user" do
-    create_current_auction
-
-    # seeing auction list
-    visit "/"
-    expect(page).to have_content(@auction.title)
-    expect(page).to have_content("Current bid:")
-
-    # going to the auction detail page
-    click_on(@auction.title)
-    page.find("a[href='#{@auction.issue_url}']")
-
-    # going via another link
-    visit "/"
-    click_on("View details »")
-    page.find("a[href='#{@auction.issue_url}']")
-
-    # logging in via bid click
-    click_on("BID")
-    expect(page).to have_content("Authorize with GitHub")
-    click_on("Authorize with GitHub")
-    # completing user profile
-    fill_in("user_duns_number", with: "123-duns")
-    click_on('Submit')
-
-    # bad ui brings us back to the home page :(
-    click_on("Bid »")
-    expect(page).to have_content("Current bid:")
-
-    # fill in the form
-    fill_in("bid_amount", with: '800')
-    click_on("Submit")
-
-    # returns us back to the bid page
-    expect(page).to have_content("Current bid:")
-    expect(page).to have_content("$800.00")
+      expect(page).to_not have_content('Bid »')
+    end
+
+    scenario "There are several auctions" do
+      # Test ordering here
```

Comment:
I think it is automatically skipped and marked if it is empty?

On Mon, Jan 4, 2016 at 2:32 PM, Alan deLevie <notifications@github.com>
wrote:

> In spec/features/bidder_interacts_with_auction_spec.rb
> <https://github.com/18F/micropurchase/pull/186#discussion_r48769848>:
>
> > -    # bad ui brings us back to the home page :(
> > -    click_on("Bid »")
> > -    expect(page).to have_content("Current bid:")
> > -
> > -    # fill in the form
> > -    fill_in("bid_amount", with: '800')
> > -    click_on("Submit")
> > -
> > -    # returns us back to the bid page
> > -    expect(page).to have_content("Current bid:")
> > -    expect(page).to have_content("$800.00")
> > +      expect(page).to_not have_content('Bid »')
> > +    end
> > +
> > +    scenario "There are several auctions" do
> > +      # Test ordering here
>
> Sounds good. Is there a way to mark this as skipped?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/micropurchase/pull/186/files#r48769848>.
>